db/queries/items.js

```diff
@@ -0,0 +1,78 @@
+const db = require('../connection');
+
+/* Fetch Items related to a specific category */
+
+const getItem = (categoryId) => {
+  const query = `SELECT *
+  FROM items
+  JOIN categories
+  ON categories.id = categories_id
+  WHERE categories_id = $1;`
+  return db.query(query,[categoryId])
+    .then(data => {
+      return data.rows[0];
```

Comment:
I believe we need the entire array of items within `rows`, so `[0]` is not needed here.